django_rules/backends.py

```diff
@@ -17,12 +17,28 @@
 from exceptions import NonexistentPermission
 from exceptions import RulesError
 
+class RulePermCache(object):
+    @cached_property
+    def rules(self):
+        return {
+            rp.codename: rp
+            for rp in  RulePermission.objects.select_related('content_type').all()
```

Comment:
```suggestion
            for rp in RulePermission.objects.select_related('content_type').all()
```
the nittiest of nits, just dropping this extra space